tools/localization_analysis/scripts/make_map.py

```diff
@@ -104,10 +91,35 @@ def make_map(
 
         # Link maps directory since conversion to BRISK map needs
         # image files to appear to be in correct relative path
-        os.symlink(maps_directory, "maps")
-        maps_bag_images = os.path.join("maps", bag_images_dir)
-        if not os.path.isdir(maps_bag_images):
-            os.symlink(bag_images, maps_bag_images)
+        build_map_command = (
```

Comment:
Can you add some details here? Was the symlink messing up sometimes?

Reply:
I'm really pulling my memory here, but if I remember correctly the problem was that if you symlink to `maps`, you're changing the dir name, so the relative path didn't work?

We're still doing very similar symlinks, in line 117 forward, the difference is that I'm getting the path location from the map itself instead of having the user specify it manually (using the build_map -info command)